test/compiler/flowgraph/check_runaway.spec.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var expect = require('chai').expect;
+var juttle_test_utils = require('../../runtime/specs/juttle-test-utils');
+var check_juttle = juttle_test_utils.check_juttle;
+
+describe('Runaway program detection', function() {
+
+    function check(juttle, is_runaway) {
+        it(`${juttle} // is ` + (is_runaway ? '' : 'not') + ' a runaway', () => {
```

Comment:
It's a little déclassé to put `it` inside a function like this, since it means you can't `it.skip` or `it.only` any of these tests.